src/CSS/Article.css

```diff
@@ -53,4 +54,44 @@
 label{
     background: none !important;
     margin-left: -10px;
+    padding: 6px 3px;
```

Comment:
Can you use a different styling element? I feel there's going to be unforeseen style problems when you use this one.

Reply:
@ssewilliam